src/snprintf.cxx

```diff
@@ -69,19 +69,29 @@ static inline
 int
 vftprintf (std::FILE * file, tchar const * fmt, std::va_list args)
 {
+    int ret;
+
+    va_list args_copy;
+
+    va_copy(args_copy, args);
```

Comment:
I am not convinced any of these are necessary. With your patch we already copy the `args` in caller.